api/openapi-spec/conventions-server..yaml

```
@@ -0,0 +1,133 @@
+openapi: 3.0.0
+info:
+  title: Conventions server 
+  description: >-
+    "a sample conventions server"
+  version: 1.0.0
+  license:
+    name: Apache-2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0.html'
+paths:
+  /webhook:
+    post:
+      description: "a sample conventions server"
+      responses:
+        200:
+          description: Successfully applied defined conventions
+          content:
+            "application/json":
+              schema:
+                $ref: '#/components/schemas/PodConventionContext'
+        400:
+          description: Error decoding PodConventionContext from the request body or request body is nil 
+        500:
+          description: Internal server error
+          content:
+            "application/json":
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+components:
+  schemas:
+    PodConventionContext: 
+      type: object
+      properties:
+        apiVersions: 
+          type: string
+        kind: 
+          type: string
+        metadata:
+          type: array 
+          items: 
+            type: string
```

the real shape of the metadata field here is the same as within the PodTemplateSpec. In practice, we only use the name field.

```suggestion
        metadata:
          type: object 
          properties: 
            name:
              type: string
```